autorest/adal/msi_linux.go

```diff
@@ -0,0 +1,4 @@
+// +build !windows
+
+// msiPath is the path to the MSI Extension settings file (to discover the endpoint)
+var	msiPath = "/var/lib/waagent/ManagedIdentity-Settings"
```

Comment:
Because we're protected behind a build tag, you may as well make this a `const` as the Unix version of this isn't dynamic.